templates/cloud/private-cloud.html

```diff
@@ -0,0 +1,483 @@
+			</p>
+		</div>
+	</div>
+</section>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-8">
+			<h2>Types of private cloud</h2>
+			<p class="u-sv3">Depending on where the private cloud runs and who manages it, the following types of private cloud can be distinguished:</p>
+		</div>
+    <div class="row u-equal-height">
+			<div class="col-4">
+				<div class="u-sv1 u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fdf83d49-Server.svg",
+            alt="On-prem",
+            width="58",
+            height="70",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+					</div>
+					<h4>On-prem</h4>
+					<p class="u-sv3">Private cloud - in this case the cloud is deployed on the organisation's premises, inside of their own data centre. The customer provides hardware and hosting facilities, and usually takes care of the private cloud maintenance and operations. On-prem private clouds are best for big enterprises who have existing physical infrastructure and enough human resources to manage the cloud.</p>
+					<p><a href="/openstack">More about on-prem cloud&nbsp;&rsaquo;</a></p>
+				</div>
+			<div class="col-4">
+				<div class="u-sv1 u-hide--small">
+				{{ image (
+					url="https://assets.ubuntu.com/v1/c5b21ec2-Private+Cloud.svg",
+					alt="Hosted",
+					width="96",
+					height="70",
+					hi_def=True,
+					loading="lazy"
+					) | safe
+				}}
+				</div>
+				<h4>Hosted</h4>
+				<p class="u-sv3">Private cloud - in this case the private cloud runs in a data centre that is outside the organisation’s facilities. The customer is still responsible for providing the hardware, but all hosting facilities (space, electricity, cooling, etc.) are delivered by the service provider. Hosted private clouds are best for organisations who do not have their own data centre and are not going to build one in the foreseeable future.</p>
+				<p><a href="/openstack/what-is-openstack">More about management software&nbsp;&rsaquo;</a></p>
+			</div>
+			<div class="col-4">
+				<div class="u-sv1 u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/3b176ef1-managed-private-cloud.svg",
+            alt="Managed",
+            width="70",
+            height="70",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+				</div>
+				<h4>Managed</h4>
+				<p>Private cloud - the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of nodes, because they do not require hiring a dedicated operations team.</p>
```

Comment:
```suggestion
				<p>Private cloud &mdash; the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of nodes, because they do not require hiring a dedicated operations team.</p>
```